cpp/velox/compute/WholeStageResultIterator.cc

```diff
@@ -363,6 +366,12 @@ std::unordered_map<std::string, std::string> WholeStageResultIterator::getQueryC
     configs[velox::core::QueryConfig::kSpillableReservationGrowthPct] =
         getConfigValue(confMap_, kSpillableReservationGrowthPct, "25");
     configs[velox::core::QueryConfig::kSpillCompressionKind] = getConfigValue(confMap_, kSpillCompressionKind, "lz4");
+    configs[velox::core::QueryConfig::kSparkBloomFilterExpectedNumItems] =
+        getConfigValue(confMap_, kBloomFilterExpectedNumItems, "1000000");
+    configs[velox::core::QueryConfig::kSparkBloomFilterNumBits] =
+        getConfigValue(confMap_, kBloomFilterNumBits, "8388608");
+    configs[velox::core::QueryConfig::kSparkBloomFilterMaxNumBits] =
```

Comment:
It seems the default value of `kSparkBloomFilterMaxNumBits `in vanilla spark is 67108864L [here ](https://github.com/apache/spark/blob/f6038302dd615f4bf9bed9c4af3d04426f7e5c5e/sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala#L434). Why here is 4194304?

Reply:
This is because velox has below requirement: https://github.com/facebookincubator/velox/blob/73d4279a14744cf4d038d3a967a49dcddbad9d39/velox/core/QueryConfig.h#L632C5-L632C24